hack/quickstart/init-master.sh

```diff
@@ -39,13 +39,11 @@ function init_master_node() {
     systemctl stop update-engine; systemctl mask update-engine
 
     etcd_render_flags=""
-    etcd_start_flags=""
 
     # Start etcd.
     if [ "$SELF_HOST_ETCD" = true ] ; then
         echo "WARNING: THIS IS NOT YET FULLY WORKING - merely here to make ongoing testing easier"
-        etcd_render_flags="--etcd-servers=http://10.3.0.15:2379 --experimental-self-hosted-etcd"
-        etcd_start_flags="--etcd-server=http://${COREOS_PRIVATE_IPV4}:12379 --experimental-self-hosted-etcd"
+        etcd_render_flags="--etcd-servers=http://10.3.0.15:2379 --experimental-self-hosted-etcd"  #TODO(diegs): maybe add --bootstrap-etcd-server=http://${COREOS_PRIVATE_IPV4}:12379 
```

Comment:
Sorry this is for the render side (not start). But I do think we should probably just pick the service IP for the user if they're using self-hosted etcd. It has to be within the service-cidr, and we already assume certain IPs for other services (e.g. DNS gets the .10).

That change can be follow-up/optimization though (maybe just add a TODO if not addressed here)